app/uk/gov/hmrc/tradergoodsprofilesrouter/models/request/eis/accreditationrequests/RequestAccreditation.scala

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
I think this should be in the `uk.gov.hmrc.tradergoodsprofilesrouter.models.request` package since it is the router's own request. It should be in the same folder as `RemoveRecordRequest` `UpdateRecordRequest` and `CreateRecordRequest`